compiler/plugins/target/AMD-AIE/iree-amd-aie/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ iree_cc_library(
   DEPS
     iree::compiler::Dialect::HAL::Target
     iree::compiler::PluginAPI
+    iree::target::amd-aie::IR::AMDAIEDialect
```

Comment:
It's unclear to me what the goal of this dialect is... and I see a huge opportunity for confusion with the existing dialects.  Can you elaborate?

Reply:
Another naming option might be "XDNA"?

Reply:
Hi @stephenneuendorffer .

Like `lowering_config` is attached to `iree_codegen` dialect, the goal was to do the same with the new `packing_config` attribute. Since we're building this as a plugin, I cannot directly use `iree_codegen` dialect.

And since `packing_config` is only used by amd-aie, it made sense to have it attached to a dialect specific to this stage (which can be renamed to `XDNA` as you suggested).

Reply:
Thanks, I understand :)

Reply:
Is it worth a comment that this should eventually be merge into `iree_codegen`?

Reply:
That's a fair ask - although I'm not sure if we would indeed want `packing_config` as part of `iree_codegen` dialect eventually since no other backend seems to be using a pack based lowering.